### PR TITLE
Update sample to 1.1.0-alpha01. See https://developer.android.com/jetpack/androidx/releases/health-connect#1.1.0-alpha01 for more details.

### PR DESCRIPTION
**Pull request by cy245**

### CODE REVIEW

File: health-connect/HealthConnectSample/app/build.gradle [hunk 1]
```diff
@@ -20,12 +20,12 @@ plugins {
 }
 
 android {
-    compileSdk 33
+    compileSdk 34
```
Comment (ithinkihaveacat) [inline health-connect/HealthConnectSample/app/build.gradle:23]: Suggest upgrading the Android Gradle Plugin if bumping this to 34. (My AS complains the existing version is not compatible with 34.)

**Review by ithinkihaveacat — APPROVED** [review]
LGTM

Suggest linking to https://developer.android.com/jetpack/androidx/releases/health-connect#1.1.0-alpha01 from the PR description for neatness and to tie things together.

File: health-connect/HealthConnectSample/app/build.gradle (hunk 1 above)
Comment (cy245) [reply health-connect/HealthConnectSample/app/build.gradle:23]: Good point - done!

**Review by ithinkihaveacat — APPROVED** [review]
LGTM

**Pull request merged**